active/progress/progress.md

```diff
@@ -0,0 +1,107 @@
+docs.microsoft.com (http://docs.microsoft.com/uwp/toolkits/winui/).
+Hopefully we'll be able to copy it mostly verbatim.
+So the second audience is everyone that reads there to learn how
+and why to use this API. -->
+
+# Background
+<!-- Use this section to provide background context for the new API(s) 
+in this spec. -->
+
+<!-- This section and the appendix are the only sections that likely
+do not get copied to docs.microsoft.com; they're just an aid to reading this spec. -->
+
+<!-- If you're modifying an existing API, included a link here to the
+existing page(s) -->
+
+<!-- For example, this section is a place to explain why you're adding this API rather than
+modifying an existing API. -->
+
+<!-- For example, this is a place to provide a brief explanation of some dependent
+area, just explanation enough to understand this new API, rather than telling
+the reader "go read 100 pages of background information posted at ...". -->
+
+>See proposal documents for:
+>
+> [determinate mode for ProgressRing](https://github.com/microsoft/microsoft-ui-xaml/issues/688) 
+>
+> [ProgressRing style](https://github.com/microsoft/microsoft-ui-xaml/issues/837)
+>
+> [Guidance for Progress controls](https://github.com/microsoft/microsoft-ui-xaml/issues/880)
+
+Progress controls indicate to a user that an operation is occuring. This includes ProgressBar and ProgressRing controls. ProgressBar has both a determinate and indeterminate state. ProgressRing only has an indeterminate state.
+
+Important APIs: [ProgressBar](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.progressbar) and [ProgressRing](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.progressring)
+# Description
+<!-- Use this section to provide a brief description of the feature.
+For an example, see the introduction to the PasswordBox control 
+(http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box). -->
+The ProgressRing control indicates that an operation is ongoing. The determinate mode of ProgressRing shows the percentage completed of a task. 
+
+The style updates to progress controls will align with new designs and with other Microsoft products.
+
+# Examples
+<!-- Use this section to explain the features of the API, showing
+example code with each description. The general format is: 
+  feature explanation,
+  example code
+  feature explanation,
+  example code
+  etc.-->
+  
+<!-- Code samples should be in C# and/or C++/WinRT -->
+
+<!-- As an example of this section, see the Examples section for the PasswordBox control 
+(https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box#examples). -->
+
+The following examples show how to use the isIndeterminate property to change the mode of the ProgressRing and the Value property to change the proportionate amount indicated. 
+
+## Indeterminate ProgressRing
+
+```xml
+<ProgressRing IsActive="True" Height="100" Width="100" IsIndeterminate="True"/>
```

Comment:
The way this is written, there's an implication that the default value for IsIndeterminate is False. Is that true? If so, will that break existing ProgressRing uses that take for granted its indeterminate nature?

Reply:
This is a good point - I'll add this as an open question. The way this is written is intended to align with ProgressBar.

Reply:
I think it would also be reasonable to prioritize compatibility with existing ProgressRing behavior over alignment with ProgressBar, i.e. make the default IsIndeterminate="False"